frame/staking/src/tests.rs

```diff
@@ -1060,22 +1082,22 @@ fn bond_extra_and_withdraw_unbonded_works() {
 		);
 
 		// Attempting to free the balances now will fail. 2 eras need to pass.
-		Staking::withdraw_unbonded(Origin::signed(10)).unwrap();
+		assert_ok!(Staking::withdraw_unbonded(Origin::signed(10), u32::max_value()));
```

Comment:
I see that you are trying to prevent failing tests here since any value greater than the correct number of spans is okay, but I think this is a bit too sloppy. A small function with a short name like `span_count_of(&10)` and passing a correct value would be better imo.

Reply:
fixed